### PR TITLE
Minor update to limit clause description

### PR DESCRIPTION
**Pull request by KRVPerera**
## Purpose
```diff
-The `limit` clause limits the number of results from the earlier clauses.
+The `limit` clause limits the number of results from the query.
```

### CODE REVIEW

**Review by praneesha — APPROVED** [review]

File: examples/limit-clause/limit_clause.description [hunk 1]
```diff
@@ -1 +1 @@
-// The `limit` clause limits the number of results from the earlier clauses.
+// The `limit` clause limits the number of results from the query.
```
Comment (MaryamZi) [inline examples/limit-clause/limit_clause.description:1]: Is this change correct? A query expression can have multiple limit clauses? @gimantha

Comment (KRVPerera) [reply examples/limit-clause/limit_clause.description:1]: limits the frames (results) from the query.

Comment (MaryamZi) [reply examples/limit-clause/limit_clause.description:1]: Limits them for the next clause, right? Not necessarily for the entire query expression?

Comment (KRVPerera) [reply examples/limit-clause/limit_clause.description:1]: I got it. There can be multiple limit clauses.

**Pull request merged**